app/logical/sources/strategies/pixiv.rb

```diff
@@ -162,7 +162,9 @@ def artist_name
       end
 
       def other_names
-        [artist_name, moniker].compact.uniq
+        other_names = [artist_name]
+        other_names << moniker unless moniker&.starts_with?("user_")
+        other_names.compact.uniq
```

Comment:
The code golfer in me would use `[artist_name, moniker].compact.uniq.grep_v(/\Auser_/i)`